DQMOffline/Trigger/python/VBFSUSYMonitor_Client_cff.py

```diff
@@ -0,0 +1,42 @@
+import FWCore.ParameterSet.Config as cms
+from DQMServices.Core.DQMEDHarvester import DQMEDHarvester
+
+vbfsusyEfficiency = DQMEDHarvester("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/SUSY/VBF/*"),
+    verbose        = cms.untracked.uint32(0), # Set to 2 for all messages
+    resolution     = cms.vstring(),
+    efficiency     = cms.vstring(
+        "effic_ht           'HT turnON;             PF HT [GeV]; efficiency'      ht_numerator           ht_denominator",
+        "effic_ht_variable  'HT turnON;             PF HT [GeV]; efficiency'      ht_variable_numerator  ht_variable_denominator",
+        "effic_met          'MET turnON;            PF MET [GeV]; efficiency'     met_numerator          met_denominator",
```

Comment:
It is okay to drop effic_ht and effic_met.
Do I need to update the code and git commit/push again?